From 66291c3c7739dd3983aa747d82885642df26a8d7 Mon Sep 17 00:00:00 2001
From: Serguei Katkov <serguei.i.katkov@intel.com>
Date: Fri, 8 Nov 2013 17:22:19 +0700
Subject: Dalvik: Disable PCG for haswell configuration

BZ: AXIA-4831

We need to switch off PCG building for haswell because
PCG depends on libsvml library which is a part of IPP
and is not built by default in haswell configuration.

If needed we will enable PCG build later.

Category: device-enablement
Domain: AOSP-Dalvik-Runtime
Origin: internal
Upstream-Candidate: no, needs rework

Change-Id: Iddcf7dc381e69db973fccc215b472f8b471d14db
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 Android.mk |   10 +++++++++-
 1 files changed, 9 insertions(+), 1 deletions(-)

diff --git a/Android.mk b/Android.mk
index f156a3d..11ee273 100644
--- a/Android.mk
+++ b/Android.mk
@@ -42,7 +42,15 @@ endif
 ifeq ($(WITH_JIT),true)
   ifeq ($(TARGET_ARCH),x86)
     ifeq ($(WITH_PCG),)
-        WITH_PCG := true
+      ifeq ($(TARGET_ARCH_VARIANT),haswell)
+        WITH_PCG := false
+      else
+        ifeq ($(TARGET_ARCH_VARIANT),silvermont)
+          WITH_PCG := false
+        else
+          WITH_PCG := true
+        endif
+      endif
     endif
   endif
 endif
-- 
1.7.4.1

